packages/angular/src/builders/webpack-browser/webpack-browser.impl.ts

```diff
@@ -31,7 +51,8 @@ function buildApp(
       return buildAppWithCustomWebpackConfiguration(
         delegateOptions,
         context,
-        pathToWebpackConfig
+        pathToWebpackConfig,
+        pathToIndexFileTransformer ?? null
```

Comment:
Is this check needed? Expected type is `string`, should it be `string | null` or `string | undefined`, like `pathToIndexFileTransformer?: string`? Since you're already checking if it has value on line 68 and 98.